metrics/cli/test/services/sentry/adapter/sentry_cli_service_adapter_test.dart

```diff
@@ -190,52 +187,34 @@ void main() {
     );
 
     test(
-      ".createRelease() prompts the user to enter the release name",
+      ".getSentryRelease() prompts the user to enter the release name",
       () async {
```

Comment:
Should we remove the `async` keyword since this function seems to be not asynchronous? Please, apply this suggestion to all similar tests.